test/AbstractHandle_server_test.py

```diff
@@ -157,15 +159,20 @@ def test_persist_handle_ok(self):
         self.assertEqual(new_counter, counter + 1)
         handles = handler.fetch_handles_by(self.ctx, {'elements': [hid], 'field_name': 'hid'})[0]
         self.assertEqual(len(handles), 1)
-        handle = handles[0]
+        handle_ret = handles[0]
         self.assertEqual(hid, 'KBH_' + str(new_counter))
-        self.assertEqual(handle.get('hid'), hid)
-        self.assertEqual(handle.get('id'), 'id')
-        self.assertEqual(handle.get('file_name'), 'file_name')
-        self.assertEqual(handle.get('created_by'), self.user_id)
+        self.assertEqual(handle_ret.get('hid'), hid)
+        self.assertEqual(handle_ret.get('id'), 'id')
+        self.assertEqual(handle_ret.get('file_name'), 'file_name')
+        self.assertEqual(handle_ret.get('created_by'), self.user_id)
+
+        # testing persist_handle a second handle
+        hid = handler.persist_handle(self.ctx, handle)[0]
```

Comment:
Should check the hid is what you expect here

Reply:
👍